semana07/labefy/src/App.js

```diff
@@ -0,0 +1,75 @@
+import './App.css';
```

Comment:
Código limpo com padronização no nome das variáveis, indentação clara facilitando a leitura e entendimento do mesmo.
Optou por tratar possíveis erros que retornam no .catch com console.log, o que apesar de impedir que a aplicação quebre, deixará o usuário no escuro quando a ação requerida não for executada.
Criou 3 componentes para as funções de Playlist, Detalhes da Playlist e  Login  o que deixou os códigos mais enxutos.
Montou um PR bem completo, com informações úteis e prints da aplicação.
Ótimo projeto! Parabéns!